packages/terra-form-select/src/terra-dev-site/test/form-select/FilterStyle.test.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import Select from '../../../Select';
+import styles from './common/Select.test.module.scss';
+
+const cx = classNames.bind(styles);
+
+class FilterStyle extends React.Component {
```

Comment:
New examples should use hooks when possible for best practice. Hooks will help simplify this test example

Reply:
Updated the example here https://github.com/cerner/terra-core/pull/3094/commits/5c0d19afda050f822e2d6e7fcd04abefa15886b6